popup.js

```diff
@@ -12,26 +12,29 @@ var state = {
     rpcPort: "31050",
     ready: 0, // ready on 8
     readyDOM: false,
-    useDebug: false
+    useDebug: false,
+    transactionsReturned: 20
 }
 
+var scrollEventListenerAdded = false

```

Comment:
add this to the state, I don't think we should do lone global variables, maybe rename it to txScrollEventListenerAdded?